### ISSUE TITLE
Server crashing on final loading steps - Exception getting block type in world

### ISSUE THREAD

**Issue by BSilberstein** [GTNewHorizons/GT-New-Horizons-Modpack#17193]
### Your GTNH Discord Username

bailers

### Your Pack Version

GT New Horizons-2.6.1

### Your Server

Private server

### Java Version

Java 8

### Type of Server

Don't know

### Your Actions

Not the first time I have had a power outage when running the server, but this time it refuses to fully launch quoting that it may be be damage or a bug preventing certain things from loading properly. The log is referencing "Exception getting block type in world" to be one of the main issues early in the log.

### Crash Report

https://pastebin.com/1biA1hLA

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the crash still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this crash consistently by following the exact steps I described above.
- [ ] I have asked other people and they confirm they also crash by following the exact steps I described above.

**Comment by Fawnzzz** [comment]
What was the solution to this? I am having the same issue and I have tried to load all backups from the same day (all I have), they give same issue. 

https://pastebin.com/vKHjPcaE

**Comment by BSilberstein** [comment]
> What was the solution to this? I am having the same issue and I have tried to load all backups from the same day (all I have), they give same issue.
> 
> https://pastebin.com/vKHjPcaE

Unfortunately I didn't find any quick fixes and don't have a concrete answer for you. I was worried that something was corrupted because of the power outage so I made a full backup of the server files and bit by bit started deleting files to try and identify what was crashing the server.

I also tried to see if I could open the server world files on a world editor to see if I could pinpoint the blocks/chunks in question but didn't have much luck. 

I went through some direct messages with the friend I was playing with when I finally got the server to launch and it looks like I ended up deleting some files associated with the JABBA mod, I couldn't tell you specifically if it was a config file or something else in the world folder.

Hopefully this info helps trying to narrow down your problem!
